app/src/main/java/com/example/dogbook/LoginActivity/LoginActivity.java

```diff
@@ -0,0 +1,84 @@
+package com.example.dogbook.LoginActivity;
```

Comment:
you can name these modules just: `login`

So, you can remove the "Activity" suffix.  We'll also usually make them all lowercase, even if there are multiple works, for package/module names